### PR TITLE
Quick command implementation

### PR DESCRIPTION
**Pull request by peske**

### CODE REVIEW

File: FluentTerminal.App/Dialogs/CustomCommandDialog.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+                        candidate =
+                            FocusManager.FindNextElement(FocusNavigationDirection.Left, options);
+
+                        if (candidate != null && candidate is Control)
+                        {
+                            if (((Control) candidate).Name == "PrimaryButton")
+                            {
+                                ((Control) candidate).Focus(FocusState.Keyboard);
+
+                                //var key = Key.A;                    // Key to send
```
Comment (felixse) [inline FluentTerminal.App/Dialogs/CustomCommandDialog.xaml.cs]: can this be removed?

File: FluentTerminal.App/Strings/de/Resources.resw [hunk 2]
```diff
@@ -748,7 +748,7 @@ Fuzzy</comment>
     <value>Fenster ein/ausblenden</value>
     <comment>Command</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```
Comment (felixse) [inline FluentTerminal.App/Strings/de/Resources.resw]: this has been renamed to NewSshTab with #410

File: FluentTerminal.App/Strings/en/Resources.resw [hunk 3]
```diff
@@ -785,18 +786,20 @@ Fuzzy</comment>
     <comment>Command
 Fuzzy</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
-    <value>New SSH tab</value>
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```
Comment (felixse) [inline FluentTerminal.App/Strings/en/Resources.resw]: this has been renamed to NewSshTab with #410

File: FluentTerminal.App/Strings/es/Resources.resw [hunk 4]
```diff
@@ -745,7 +745,7 @@
     <value>Cambiar ventana</value>
     <comment>Command</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```
Comment (felixse) [inline FluentTerminal.App/Strings/es/Resources.resw]: this has been renamed to NewSshTab with #410

File: FluentTerminal.App/Strings/zh-CN/Resources.resw [hunk 5]
```diff
@@ -785,7 +785,7 @@ Fuzzy</comment>
     <comment>Command
 Fuzzy</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```
Comment (felixse) [inline FluentTerminal.App/Strings/zh-CN/Resources.resw]: this has been renamed to NewSshTab with #410

File: FluentTerminal.App/Views/MainPage.xaml [hunk 6]
```diff
@@ -56,7 +56,14 @@
                         </MenuFlyoutItem>
                         <MenuFlyoutItem
                             Command="{x:Bind ViewModel.AddSshShellCommand}"
-                            x:Uid="NewSshTab">
+                            x:Uid="NewRemoteTab">
```
Comment (felixse) [inline FluentTerminal.App/Views/MainPage.xaml]: this has been renamed to NewSshTab with #410

**Comment by mjs** [comment]
When I try to run the code in this PR I get an ArgumentException in the FluentTerminal.App.ViewModels.CommandItemViewModel constructor. It's getting passed a null command, possibly by FillCommandHistory.

@peske: we can go through it during our meeting. I'll try to leave my settings and profiles as they are in case that's relevant.

File: FluentTerminal.App/Strings/de/Resources.resw (hunk 2 above)
Comment (peske) [reply FluentTerminal.App/Strings/de/Resources.resw]: There's _New remote tab_ in the main menu, and the control name is `NewRemoteTab`. I don't see `NewSshTab` anywhere except in `Command` enumeration, but in this case `Text` property doesn't make any sense.
Honestly, I can't remember now if I've renamed the control to `NewRemoteTab`, but resources are coming from POEditor anyway...

File: FluentTerminal.App/Strings/en/Resources.resw (hunk 3 above)
Comment (peske) [reply FluentTerminal.App/Strings/en/Resources.resw]: Again, I don't understand what you're telling me. I don't see `NewSshTab` control anywhere in your master. Only `NewRemoteTab` control in Main.xaml.

File: FluentTerminal.App/Strings/es/Resources.resw (hunk 4 above)
Comment (peske) [reply FluentTerminal.App/Strings/es/Resources.resw]: The same answer as above...

File: FluentTerminal.App/Strings/zh-CN/Resources.resw (hunk 5 above)
Comment (peske) [reply FluentTerminal.App/Strings/zh-CN/Resources.resw]: The same answer as above...

File: FluentTerminal.App/Views/MainPage.xaml (hunk 6 above)
Comment (peske) [reply FluentTerminal.App/Views/MainPage.xaml]: @felixse , can you please fix these names as you want them to be? I don't understand...

File: FluentTerminal.App/Dialogs/CustomCommandDialog.xaml.cs (hunk 1 above)
Comment (peske) [reply FluentTerminal.App/Dialogs/CustomCommandDialog.xaml.cs]: I'll remove.

File: FluentTerminal.App/Views/MainPage.xaml (hunk 6 above)
Comment (peske) [reply FluentTerminal.App/Views/MainPage.xaml]: Ahh, OK, i'll rename it back.

File: FluentTerminal.App/Strings/de/Resources.resw (hunk 2 above)
Comment (peske) [reply FluentTerminal.App/Strings/de/Resources.resw]: Fixed.

File: FluentTerminal.App/Strings/en/Resources.resw (hunk 3 above)
Comment (peske) [reply FluentTerminal.App/Strings/en/Resources.resw]: Fixed.

File: FluentTerminal.App/Strings/es/Resources.resw (hunk 4 above)
Comment (peske) [reply FluentTerminal.App/Strings/es/Resources.resw]: Fixed.

File: FluentTerminal.App/Strings/zh-CN/Resources.resw (hunk 5 above)
Comment (peske) [reply FluentTerminal.App/Strings/zh-CN/Resources.resw]: Fixed.

File: FluentTerminal.App/Views/MainPage.xaml (hunk 6 above)
Comment (peske) [reply FluentTerminal.App/Views/MainPage.xaml]: Fixed.

**Comment by peske** [comment]
@mjs - the issue should be fixed now. Please let me know when you try.

File: FluentTerminal.App/ViewModels/CommandItemViewModel.cs [hunk 7]
```diff
@@ -82,250 +86,263 @@ public CommandItemViewModel(ExecutedCommand command)
...
-            return p;
         }
 
-        public void SetFilter(string filter, string[] filterWords)
+        /// <summary>
+        /// The purpose of this method is to find matches between filter words (<paramref name="lowercaseFilterWords"/>),
+        /// and the command words (<see cref="_commandWords"/>). I'm aware that the code is quite confusing, but it kinda
+        /// has to be...
+        /// </summary>
+        private void CalculateMatchPrivate(string[] lowercaseFilterWords)
```
Comment (felixse) [inline FluentTerminal.App/ViewModels/CommandItemViewModel.cs]: I think we should move this out of the ViewModel at some point

**Pull request merged**